public/src/store.jsx

```diff
@@ -19,6 +20,8 @@ const appReducer = combineReducers({
   form: formReducer,
   auth: authStatusReducer,
   account_info: accountInfoReducer,
+  script: scriptInfoReducer,
```

Comment:
Again, this is a duplicate, but we will have to wait for consolidation. Looks good.